src/constants.js

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+  TOOLTIP_OFFSET_Y: 10
```

Comment:
If that's only used in widgets maybe we should "namespace" it: 

```
module.exports = {
  WIDGETS: {
    TOOLTIP_OFFSET_Y: 10
  }
}
```

so it get's easier to refactor later if we have tons of constants:

```
module.exports = {
  WIDGETS: require('./widgets')
}
```